cmake/GenerateRequirementsTable.py

```diff
@@ -66,10 +73,14 @@ class Requirement:
 
     descr = None
     tix = None
+    tname = None
 
-    def __init__(self, ID, testFile, rawReqBlock):
+    def __init__(self, ID, testName, testFile, rawReqBlock):
```

Comment:
Can you add documentation for testName in Args above in the docstring?

Reply:
Addressed this, please check.